boards/riscv/it8xxx2_evb/it8xxx2_evb.dts

```diff
@@ -12,11 +12,6 @@
 	model = "IT8XXX2 EV-Board";
 	compatible = "riscv,it8xxx2-evb";
 
-	aliases {
-		pwm-led0 = &led0;
-		pwm-led1 = &led1;
-	};
```

Comment:
Does this have anything to do with adding the tach driver?  If not can you please make this a different commit?

Reply:
Ok, I will create another commit for cleanup.

Reply:
thanks

Reply:
These changes still need to be moved to a different commit

Reply:
Done